content/WebAuthn/Supporting_U2F_or_FIDO2_Security_Keys_on_iOS_or_iPadOS/FAQ.adoc

```diff
@@ -0,0 +1,39 @@
+== FAQ
+
+
+=== Question: My device, iOS version, and YubiKey support WebAuthn(FIDO2) but the key is not responding to my touch.
+
+*Answer*: If you are trying to register a key for the first time with a service, you may have encountered one of the link:Known_Issues.adoc[Known Issues] that we are working with Apple on resolving. Table 1 in link:Security_Key_Compatibility.adoc[Security Key Compatibility] shows that nearly all FIDO2/WebAuthn Authentication flows will work. *Workaround*: **Register** the YubiKey with a service on a supported **desktop** browser. All subsequent  **authentication** can be done using the same key via Safari **mobile** browser.
```

Comment:
Nit: to me the text here doesn't make an easy read, it might be the multiple bold words after Workaround.

Reply:
Right. I've removed all the bolding except for the word "workaround".